### PR TITLE
Add Mysql support 

### PR DESCRIPTION
**Pull request by Nsttt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ services:
 
 ## Running the Application with Docker
 
-There is an image available on `ghcr.io/thedevminertv/postgres_s3_backup` that you can use.
+There is an image available on `ghcr.io/thedevminertv/database_s3_backup` that you can use.
```
Comment (TheDevMinerTV) [inline README.md]:
```suggestion
There is an image available on `ghcr.io/thedevminertv/database-s3-backup` that you can use.
```

File: Dockerfile [hunk 2]
```diff
@@ -10,7 +10,7 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main .
 FROM alpine:latest
 WORKDIR /root/
 
-RUN apk --no-cache add ca-certificates postgresql-client
+RUN apk --no-cache add ca-certificates postgresql-client mysql-client 
 
 COPY --from=builder /app/main /bin/postgres-s3-backup
```
Comment (TheDevMinerTV) [inline Dockerfile]: Can you rename the binary to`database-s3-backup` please? Don't forget the `CMD` below.

File: main.go [hunk 3]
```diff
@@ -42,7 +43,7 @@ func main() {
 			log.Fatalf("Failed to parse url %s: %s", rawUrl, err)
 		}
 
-		port := 5432
+		port := 0
```
Comment (TheDevMinerTV) [inline main.go:46]: This should default to the default port of the database type.

**Review by TheDevMinerTV — COMMENTED** [review]
LGTM, but does it support MariaDB as well?

Oh, you have to rebase as well btw.

File: Dockerfile (hunk 2 above)
Comment (Nsttt) [reply Dockerfile]: Done

File: main.go (hunk 3 above)
Comment (Nsttt) [reply main.go:46]: Done

**Pull request merged**